test/test618.js

```diff
@@ -0,0 +1,33 @@
+if (typeof exports === 'object') {
+    var assert = require("assert");
+    var alasql = require('..');
+}
+
+var test = '618'; // insert test file number
+
+describe('Test ' + test + ' - Where RTRIM will remove leading whitespace characters in an expression.', function() {
```

Comment:
Can I ask you to add a few tests to see it behaves correctly with tabs and newlines?